### DISCUSSION TITLE
Build-in GGUF Loader

### DISCUSSION THREAD

**Discussion by razvanab** [comfyanonymous/ComfyUI#5685] [Ideas]
It is possible to have a GGUF loader for both the model and the clip model build-in without installing the extension?
Sometimes I just want to try a new prompt fast, so I open Comfy UI with "--disable-all-custom-nodes", but then I can't use the GGUF version of the FLUX.

**Comment by ltdrdata** [comment]
Is this a request?

**Reply by razvanab** [reply]
Yes, if it is possible to implement.

**Comment by blepping** [comment]
Why not just do something like create `custom_nodes-gguf` and `custom_nodes-real` directories (where you have your normal mods installed to `custom_nodes-real` and just ComfyUI-GGUF in `custom_nodes-gguf`) and just use a symbolic link to whichever set of custom nodes you want to use at the moment?

**Reply by razvanab** [reply]
That might work too. Thanks
